### PR TITLE
Fix list styling issues

### PR DESCRIPTION
**Pull request by devpow112**
Fixes issues after styling changes to list. Changes were introduced in https://github.com/BrightspaceUI/core/pull/2301.

**Before**
image

**After**
image

### CODE REVIEW

File: src/components/editor/rubric-criteria-editor.js [hunk 1]
```diff
@@ -118,10 +118,6 @@ export class RubricCriteriaEditor extends composeMixins(
 				[hidden] {
 					display: none !important;
 				}
-
```
Comment (devpow112) [inline src/components/editor/rubric-criteria-editor.js:121]: Not needed anymore

File: src/components/editor/rubric-structure-editor.js [hunk 2]
```diff
@@ -86,16 +86,12 @@ export class RubricStructureEditor extends composeMixins(
 					display: flex;
 					justify-content: space-between;
 					flex-wrap: wrap;
-					margin-left: var(--d2l-rubric-editor-start-gutter-width); /* ie11 fix */
```
Comment (devpow112) [inline src/components/editor/rubric-structure-editor.js:89]: ie11 is no longer supported.

File: src/styles/rubric-editor-cell-styles.js [hunk 3]
```diff
@@ -3,8 +3,8 @@ import { css } from 'lit-element/lit-element.js';
 
 export const rubricEditorCellStyles = css`
 	:host {
-		--d2l-rubric-editor-start-gutter-width: 2.1rem;
-		--d2l-rubric-editor-end-gutter-width: 2.1rem; /* trash can width = 50px including halo */
+		--d2l-rubric-editor-start-gutter-width: 2.2rem;
```
Comment (devpow112) [inline src/styles/rubric-editor-cell-styles.js:6]: Just had to bump this by `0.1rem`.

File: src/styles/rubric-editor-cell-styles.js [hunk 4]
```diff
@@ -3,8 +3,8 @@ import { css } from 'lit-element/lit-element.js';
...
+		--d2l-rubric-editor-end-gutter-width: 2.2rem; /* trash can width = 50px including halo */
```
Comment (devpow112) [inline src/styles/rubric-editor-cell-styles.js:7]: Just had to bump this by `0.1rem`.

**Review by mpharoah-d2l — APPROVED** [review]

**Pull request merged**